packages/@aws-cdk/aws-stepfunctions-tasks/lib/evaluate-expression.ts

```diff
@@ -58,7 +58,7 @@ export class EvaluateExpression extends sfn.TaskStateBase {
   constructor(scope: Construct, id: string, private readonly props: EvaluateExpressionProps) {
     super(scope, id, props);
 
-    this.evalFn = createEvalFn(this.props.runtime || lambda.Runtime.NODEJS_10_X, this);
+    this.evalFn = createEvalFn(this.props.runtime || lambda.Runtime.NODEJS_12_X, this);
```

Comment:
same here.

Reply:
Reverted and created a follow-up issue https://github.com/aws/aws-cdk/issues/13224